ovc5/scripts/install_sd.sh

```diff
@@ -74,17 +90,26 @@ EOF
   fi
 
   save_dir=$pwd
-
   cp $XSA_PATH $PETALINUX_DIR/system.xsa
   cd $PETALINUX_DIR
-  petalinux-config --get-hw-description --silentconfig
-  petalinux-build
-  petalinux-package \
-    --boot \
-    --force \
-    --fsbl $BOOT_FILES_DIR/zynqmp_fsbl.elf \
-    --fpga \
-    --u-boot
+
+  # Creating directory for petalinux boot files
+  mkdir -p $SD_BOOT_FILES_DIR
+  mkdir -p $EMMC_BOOT_FILES_DIR
+
+  echo "Building eMMC boot image"
+  sed -i 's/mmcblk1p2/mmcblk0p2/g' $PETALINUX_DIR/project-spec/configs/config
```

Comment:
I'm lowkey interested in finding out whether we can skip this step and just specify kernel boot parameters in a different way (i.e. in a text file, or in u-boot), but I honestly have no idea if it is possible.

Reply:
I did come across discussions of `uEnv.txt` during my exploration in trying to find a solution for this problem. Unfortunately, I couldn't get it to work so I gave up and made this jank solution 😞